.github/workflows/test-database.yml

```
@@ -229,11 +229,27 @@ jobs:
       - name: Validate XML
         run: |
           if grep -q "<Operations>" "report.xml"; then
-             echo
-             echo "Migration files are not in sync with the files in the Sql project. Review to make sure that any stored procedures / other db changes match with the stored procedures in the Sql project."
+             echo "ERROR: Migration files are not in sync with the SQL project"
+             echo ""
+             echo "Check these locations:"
+             echo "  - Migration scripts: util/Migrator/DbScripts/"
+             echo "  - SQL project files: src/Sql/"
+             echo "  - Download 'report.xml' artifact for full details"
+             echo ""
+             
+             # Show actual SQL differences - exclude database setup commands
+             if [ -f "diff.sql" ] && [ -s "diff.sql" ]; then
```

I think the `-f "diff.sql"` can be removed since its redundant with `-s` 
```suggestion
             if [ -s "diff.sql" ]; then
```

---

good catch! just pushed the change